oqpy/classical_types.py

```diff
@@ -445,3 +445,22 @@ def to_ast(self, program: Program) -> ast.Expression:
         if self.subroutine_decl is not None:
             program._add_subroutine(self.identifier.name, self.subroutine_decl)
         return ast.FunctionCall(self.identifier, map_to_ast(program, self.args))
+
+
+class RangeDefinition(OQPyExpression):
```

Comment:
There is already a `convert_range` function (defined earlier in this file) which does something very similar... can you use that instead of making a new class?

Reply:
`convert_range` takes python `range` as input. I still need a data type compatible with oqpy var if I integrate the code path into `convert_range`. So there will still be different code path in `convert_range`.